basilisk/gui/edit_block_dialog.py

```
@@ -0,0 +1,487 @@
+
+This module provides a dialog for editing message blocks, allowing users to modify
+prompt text, attachments, model settings, and other parameters. It reuses components
+from BaseConversation for consistency in the user interface.
+"""
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING, Optional
+
+import wx
```

Pylint (3.3.7)

[error] Unable to import 'wx'

(E0401)